### PR TITLE
[IMP] calendar: improve event colors

### PR DESCRIPTION
**Pull request by leonardopavanrocha**
In calendar events, the displayed colors are a bit dark and meeting
names in black don't have enough contrast. This commit fixes this
issue and improves general design of the calendar module.

task-2704288

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/legacy/js/views/calendar/calendar_renderer.js [hunk 1]
```diff
@@ -498,6 +498,7 @@ return AbstractRenderer.extend({
                         element.addClass('o_calendar_color_1');
                     }
                 } else {
+                console.log(self._eventRender(event))
```
Comment (arj-odoo) [inline addons/web/static/src/legacy/js/views/calendar/calendar_renderer.js:501]: ??

Comment (leonardopavanrocha) [reply addons/web/static/src/legacy/js/views/calendar/calendar_renderer.js:501]: oopsie

**Comment by arj-odoo** [comment]
@robodoo rebase-ff

**Comment by arj-odoo** [comment]
@robodoo r+

**Pull request closed without merging**